src/TwigComponent/src/ComponentRenderer.php

```diff
@@ -130,11 +130,10 @@ private function preRender(MountedComponent $mounted, array $context = []): PreR
 
             // add computed properties proxy
             ['computed' => new ComputedPropertiesProxy($component)],
-
+            $props,
             // add attributes
             [$metadata->getAttributesVar() => $mounted->getAttributes()],
-            $props,
-            ['__props' => $props]
+            ['__props' => array_merge($mounted->getInputProps(), $props)]
```

Comment:
```suggestion
            ['__props' => $props],
```

Line 120, you wrote
```php
$props = array_merge($mounted->getInputProps(), ....)
```

So here it's as if you wrote 

```php
      ['__props' => array_merge($mounted->getInputProps(), $mounted->getInputProps(), ....
```

:)